testing/rkt/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Brian Rosner <brian@brosner.com>
+# Maintainer: Brian Rosner <brian@brosner.com>
+pkgname=rkt
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="container engine for Linux designed to be composable, secure, and built on standards"
+url="https://github.com/coreos/rkt"
+arch="x86_64"
+license="ASL 2.0"
+depends=""
+depends_dev=""
+makedepends="$depends_dev coreutils grep gzip diffutils cpio go bc linux-headers wget file gnupg autoconf automake bash openssl-dev acl-dev xz sed"
```

Comment:
This package does not provide -dev subpackage, so it’s not useful to declare and use `depends_dev` variable. abuild even warns about this. Please remove it and also wrap line on 80 characters.